pkgs/os-specific/linux/nvidia-x11/settings.nix

```diff
@@ -64,6 +65,12 @@ let
     '';
   };
 
+  runtimeDependencies = [
+    libglvnd libXrandr
```

Comment:
In that case, add libXv here.

And I think these are picked up by autoPatchelfHook if you add it.